server/controllers/finance/reports/priceList/index.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * @overview server/controllers/finance/reports/financial.employee.js
+ *
+ * @description
+ * This file contains code to create a PDF report for financial activities of an employee
+ *
+ * @requires Employee
+ * @requires ReportManager
+ */
+const _ = require('lodash');
+const ReportManager = require('../../../../lib/ReportManager');
+const db = require('../../../../lib/db');
+
+const TEMPLATE = './server/controllers/finance/reports/priceList/report.handlebars';
+
+const PDF_OPTIONS = {
+  filename : 'FORM.LABELS.PRICE_LIST',
+};
+
+exports.report = (req, res, next) => {
+  const options = req.query;
+
+  let report;
+
+  _.defaults(options, PDF_OPTIONS);
+
+  // set up the report with report manager
+  try {
+    report = new ReportManager(TEMPLATE, req.session, options);
+
+    const priceListUUID = db.bid(req.params.uuid);
+
+    return lookupPriceList(priceListUUID)
+      .then((data) => report.render(data))
+      .then(result => {
+        res.set(result.headers).send(result.report);
+      })
+      .catch(next)
+      .done();
+
+  } catch (e) {
+    return next(e);
+  }
+};
+
+
+function lookupPriceList(uuid) {
+  let _priceList;
+  let sql =
+    `SELECT BUID(uuid) AS uuid, label, description, created_at, updated_at
+    FROM price_list WHERE uuid = ?;`;
+
+  return db.one(sql, [uuid])
+    .then((row) => {
+      _priceList = row;
+
+      sql =
+        `SELECT BUID(pl.uuid) as uuid, pl.label, pl.value, pl.is_percentage, pl.created_at, iv.text as inventoryLabel
+        FROM price_list_item pl
+        JOIN inventory as iv ON iv.uuid = pl.inventory_uuid
+        WHERE pl.price_list_uuid = ?;`;
```

Comment:
You have to enhance the query for having all inventory that can be sold. 

The idea behind the price list is a list of article or service that the enterprise sale and their cost computed according enterprise rate.

You could see a kind of price list without calculation when you export the inventory registry to pdf.